src/modules/launcher/Wox.Plugin/PluginPair.cs

```diff
@@ -67,10 +67,11 @@ public void Update(PowerLauncherPluginSettings setting, IPublicAPI api)
             {
                 Metadata.Disabled = false;
                 InitializePlugin(api);
+
                 if (!IsPluginInitialized)
                 {
-                    var title = string.Format(CultureInfo.CurrentCulture, Resources.FailedToLoadPluginTitle, Metadata.Name);
-                    api.ShowMsg(title, Resources.FailedToLoadPluginDescription, string.Empty, false);
+                    string description = $"{Resources.FailedToLoadPluginDescription} {Metadata.Name}\n\n{Resources.FailedToLoadPluginDescriptionPartTwo}";
+                    api.ShowMsg(Resources.FailedToLoadPluginTitle, description, string.Empty, false);
```

Comment:
What about third-party plugins? I had a version showing the author name and web link from plugin json. But then some built-in plugins are show links to authors github page. It is a bit tricky.

Any ideas?

Reply:
Good point. Hm...

Well, if it's the plugin we support - we'd like a bug report and an issue opened. If it's a third-party plugin, we can't do much there.

I believe the majority of users only use plugins that we ship. So it might make sense to ask for bug report anyway. In 1 in X cases when it's third-party plugin, we can just say it's not our plugin and we can do much there 🤷

Reply:
That sounds good to me!